pkg/webhook/workspace/config.go

```diff
@@ -31,6 +32,11 @@ func Configure(ctx context.Context) error {
 		return err
 	}
 
+	namespace, err := k8sutil.GetOperatorNamespace()
```

Comment:
Just wondering if it makes sense to evaluate it once and return from Config object.